src/engine/renderer/tr_local.h

```diff
@@ -1207,6 +1207,8 @@ enum class deluxeMode_t { NONE, GRID, MAP };
 
 		bool            dpMaterial;
 
+		bool shaderHasNoLight;
```

Comment:
This boolean is only used to know there is no light to do only do things when there is no light.
This is used to cancel overbright when there is no light, since overbright only applies to lights.